### PR TITLE
Charts API reference, theme reference and API explorer docs pages

### PR DESCRIPTION
**Pull request by manapeirov**

### CODE REVIEW

File: grid-packages/ag-grid-docs/documentation/doc-pages/charts-api-explorer/index.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 title: "API Explorer"
 ---
 
+This explorer can be used to gain familiarity with the AG Charts API and see in real-time how different `options` affect charts. A code snippet showing the use of the API is produced based on the point and click configurations specified in the explorer.
```
Comment (alantreadway) [inline grid-packages/ag-grid-docs/documentation/doc-pages/charts-api-explorer/index.md:5]: Nit: Maybe `The API explorer` rather than `This explorer`?

File: grid-packages/ag-grid-docs/documentation/doc-pages/charts-api-themes/index.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
 title: "Themes Reference"
 ---
 
-This page documents the AG Charts Themes API. You can find more details about AG Charts themes in the [Themes](/charts-themes/) section.
+Themes are used for configuring the chart appearance. They can help simplify the chart `options` and avoid duplication if a consistent style is required across multiple charts.
```
Comment (alantreadway) [inline grid-packages/ag-grid-docs/documentation/doc-pages/charts-api-themes/index.md:5]: Nit: `the chart appearance.` seem wrong, perhaps `your charts apperance` or `a charts apperance`?

File: grid-packages/ag-grid-docs/documentation/doc-pages/charts-api/index.md [hunk 3]
```diff
@@ -2,11 +2,19 @@
 title: "Options Reference"
 ---
 
-This page documents the AG Charts API. You can find more details about getting started with AG Charts in the [Getting Started](/charts-getting-started/) section. You can also explore the API and see in real-time how different options affect charts using the [API Explorer](/charts-api-explorer/).
+The `AgChartOptions` interface is the gateway to creating charts, it includes all the configuration options available in AG Charts.
```
Comment (alantreadway) [inline grid-packages/ag-grid-docs/documentation/doc-pages/charts-api/index.md:5]: I would reword this a bit perhaps?

> it includes all the configuration options available in AG Charts.

...implies to me that there are other ways to perform configuration perhaps? (But we aren't promoting those anymore)

File: grid-packages/ag-grid-docs/documentation/doc-pages/charts-api-explorer/index.md (hunk 1 above)
Comment (alantreadway) [reply grid-packages/ag-grid-docs/documentation/doc-pages/charts-api-explorer/index.md:5]: NOTE: If we have some intro text added, I wonder if we should add a `API Explorer` heading in the blue bar at the top of the explorer to delineate the enclosing page from the explorer?

File: grid-packages/ag-grid-docs/documentation/doc-pages/charts-api/index.md (hunk 3 above)
Comment (manapeirov) [reply grid-packages/ag-grid-docs/documentation/doc-pages/charts-api/index.md]: I wasn't sure how to rephrase this, I took it out as it wasn't adding any value

File: grid-packages/ag-grid-docs/documentation/doc-pages/charts-api-themes/index.md (hunk 2 above)
Comment (manapeirov) [reply grid-packages/ag-grid-docs/documentation/doc-pages/charts-api-themes/index.md]: Changed

File: grid-packages/ag-grid-docs/documentation/doc-pages/charts-api-explorer/index.md (hunk 1 above)
Comment (manapeirov) [reply grid-packages/ag-grid-docs/documentation/doc-pages/charts-api-explorer/index.md]: Changed the wording and added the heading

**Review by alantreadway — APPROVED** [review]
LGTM - thanks for these updates!
